cg/meta/compress/compress.py

```diff
@@ -252,7 +270,9 @@ def update_fastq_hk(
 
     def add_fastq_hk(self, sample_id: str, fastq_first: Path, fastq_second: Path) -> None:
         """Add FASTQ files to housekeeper"""
-        fastq_tags = [sample_id, "fastq"]
+
+        fc_name: str = self.get_flow_cell_name(fastq_path=fastq_first)
```

Comment:
```suggestion
        flow_cell_name: str = self.get_flow_cell_name(fastq_path=fastq_first)
```
Nice to be explicit - then I do not have to remember what fc stands for and you have autocomplete anyway in your editor

Reply:
done